### ISSUE TITLE
test_query/p2703_s3526.jpg has 10 captions.

### ISSUE THREAD

**Issue by bunKiatIunn** [ShuangLI59/Person-Search-with-Natural-Language-Description#5]
![default](https://user-images.githubusercontent.com/43326579/53241290-3320bf80-36dc-11e9-8b5b-27457f455ceb.png)

**Comment by ShuangLI59** [comment]
Yes, only several images have more than 2 descriptions, you can use the first two sentences. This won't influence the final results much.

**Comment by miracle24** [comment]
Hello. I found in caption_all.json, no split information is provided for each image. @bunKiatIunn @ShuangLI59 Do you know how to split the dataset into the training set, validation set, and test set?
